specification/containerinstance/resource-manager/Microsoft.ContainerInstance/2017-08-01-preview/containerInstance.json

```diff
@@ -41,7 +41,8 @@
             "$ref": "./examples/ContainerGroupsList.json"
           }
         },
-        "description": "Get the list of container groups in a given subscription.",
+        "summary": "Get a list of container groups in the specified subscription.",
+        "description": "Get a list of container groups in the specified subscription. This operation returns properties of each container group including containers, image registry credentials, restart policy, IP address type, OS type, state, and volumes.",
```

Comment:
Added each property of the container group to conform to how ACS describes each service.